api/src/main/java/org/eclipse/microprofile/metrics/annotation/package-info.java

```diff
@@ -70,17 +70,17 @@
  * </pre>
  *
  *
- * <h2>CDI Qualifier</h2>
+ * <h2>RegistryScope annotation</h2>
  * <p>
- * The {@link org.eclipse.microprofile.metrics.annotation.RegistryType RegistryType} is used to identify which
- * <code>MetricRegistry</code> (Application, Base, or Vendor) should be injected. By default, no
- * <code>RegistryType</code> will inject the application <code>MetricRegistry</code>.
+ * The {@link org.eclipse.microprofile.metrics.annotation.RegistryScope RegistryScope} is used to identify which
+ * <code>MetricRegistry</code> (Application, Base, Vendor or a user-defined scope) should be injected. By default, no
+ * <code>RegistryScope</code> will inject the application <code>MetricRegistry</code>.
```

Comment:
I should have caught this before. As written, this _could_ be interpreted to mean that "there is no RegistryScope annotation which will inject the application registry." (An unlikely interpretation, but a possible one.) 

Maybe change the sentence "By default..." to something like "Injecting a <code>MetricRegistry</code> without a <code>RegistryScope</code> annotation gives the application-scoped <code>MetricRegistry</code>."